common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -43,6 +44,48 @@ public class CommonValue {
             + "          \"custom_attribute_names\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}}\n"
             + "        }\n"
             + "      }\n";
+    public static final String ML_MODEL_GROUP_INDEX_MAPPING = "{\n" +
```

Comment:
@rbhavna I don't see `tags` field in `ML_MODEL_GROUP_INDEX_MAPPING`. As tags will be key value pairs, suggest take a look at flat object